holmes/core/tool_calling_llm.py

```
@@ -322,6 +322,26 @@ def _execute_tool_decisions(
                             result="OAuth authentication failed. Please try again.",  # type: ignore
                         )
 
+                if tool_decision.edit_command is not None:
+                    try:
+                        edited_params = json.loads(tool_call.function.arguments or "{}")
+                    except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)